src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -118,12 +119,15 @@ public function send(string $body, array $headers, int $delay = 0): string
                 'available_at' => ':available_at',
             ]);
 
-        $this->executeQuery($queryBuilder->getSQL(), [
+        $this->driverConnection->executeQuery($queryBuilder->getSQL(), [
             ':body' => $body,
             ':headers' => json_encode($headers),
             ':queue_name' => $this->configuration['queue_name'],
-            ':created_at' => self::formatDateTime($now),
```

Comment:
please remove the `formatDateTime` method

Reply:
Can we do this in a bugfix release? After all, the method is declared public. 😕

Reply:
I was thinking I delete it into another P.R. for the 4.4 branch to do not do a BC Break?

Reply:
can be removed according to https://github.com/symfony/symfony/pull/32640#discussion_r305633648